exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,40 @@
+function bracket(str){
+var paren = ['(', ')']
+var curly = ['{','}']
+var square = ['[',']']
+var closings = [')', '}', ']']
+
+var arr = str.split('') //['{','}']
```

Comment:
I tried testing the code and for some reason it tells me that ".split()" is not a function. If might be because of my test or maybe it's because "str" in line 7 is not read as string.